src/main/java/frc/robot/subsystems/PivotSubsystem.java

```diff
@@ -22,13 +27,18 @@
  * @returns pivotPosition through the getPosition()
  */
 
-public class PivotSubsystem extends SubsystemBase {
+public class PivotSubsystem extends PIDSubsystem {
   /** Creates a new PivotSubsystem. */
 
   private CANSparkMax pivotMotor;
 
   public PivotSubsystem() {
    
+    super(
+       // The controller that the command will use
```

Comment:
format this file

Reply:
What do you mean by format?

Reply:
I think the indentation is off. Do `CTRL+SHIFT+P` and search for `format`